misk/src/main/kotlin/misk/tracing/MiskTracer.kt

```diff
@@ -0,0 +1,19 @@
+package misk.tracing
+
+import io.opentracing.Tracer
+import javax.inject.Inject
+
+class MiskTracer @Inject internal constructor(
```

Comment:
nit: Don't preface classes with ``Misk`` unless absolutely necessary.

Reply:
Only doing it here because there are so many other overloaded uses of simply `Tracer`